### DISCUSSION TITLE
Loading mutliple custom models

### DISCUSSION THREAD

**Discussion by asktimfisher** [dscripka/openWakeWord#81] [General]
Wow, noob question...

I have successfully trained 3 models. I can spin them all up individually just fine:

`wakeword_models=["my_model.tflite"]`

But I want to listen for them all at once. How do I do that?

**Comment by dscripka** [comment]
You can load multiple models by providing multiple file paths to the `wakeword_models` argument. For example:

```python
wakeword_models =["my_model1.tflite", "my_model2.tflite"]
```

When loaded like this, the dictionary returned by the `predict` function will have a key for each loaded model.

**Reply by asktimfisher** [reply]
Thank you!
